extensions/adobe/experience/ajo/loyalty/loyalty.challenge.task.schema.json

```diff
@@ -0,0 +1,180 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/ajo/loyalty/challenge/task",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Loyalty Challenge Task",
+  "description": "Details about a task within a loyalty challenge, including state, dates, variables, and completion tracking.",
+  "type": "object",
+  "meta:extensible": true,
+  "definitions": {
+    "task": {
+      "properties": {
+        "https://ns.adobe.com/experience/ajo/loyalty/state": {
+          "title": "Task State",
+          "type": "string",
+          "description": "Current state of the task.",
+          "minLength": 1
+        },
+        "https://ns.adobe.com/experience/ajo/loyalty/name": {
+          "title": "Task Name",
+          "type": "string",
+          "description": "Name of the task.",
+          "minLength": 1
+        },
+        "https://ns.adobe.com/experience/ajo/loyalty/description": {
+          "title": "Task Description",
+          "type": "string",
+          "description": "Description of the task.",
+          "minLength": 1
+        },
+        "https://ns.adobe.com/experience/ajo/loyalty/startDate": {
+          "title": "Task Start Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time when the task started."
+        },
+        "https://ns.adobe.com/experience/ajo/loyalty/endDate": {
+          "title": "Task End Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time when the task ended."
+        },
+        "https://ns.adobe.com/experience/ajo/loyalty/variables": {
+          "title": "Variables",
+          "type": "object",
+          "description": "Task variables and configuration.",
+          "properties": {
+            "https://ns.adobe.com/experience/ajo/loyalty/include": {
```

Comment:
Use field name as `xdm:include` here, otherwise you will see it getting nested under _experience.ajo.loyalty object.